app/views/user_mailer/staying_on_message.html.erb

```diff
@@ -0,0 +1,68 @@
+<p>Hi <%= @user.first_name %>,</p>
+<p>
+  Many of your reached out to us yesterday with the question 'What about Philadelphia?' We were
+  certainly remiss in not addressing the convention. In fact, part of our decision to close down
+  the site was because of the disconnect in available hosts and supporters trying to get to the
```

Comment:
Changed "disconnect in available hosts and supporters" to 'small number in available hosts vs. large number of guests".
Support pick "supporter" or "guest" and use it consistently.